sycl/test-e2e/AddressSanitizer/lit.local.cfg

```
@@ -10,9 +10,9 @@ config.substitutions.append(
 
 config.unsupported_features += ['cuda', 'hip']
 
-# FIXME: Skip some of gpu devices, waiting for gfx driver uplifting
-config.unsupported_features += ['gpu-intel-gen9', 'gpu-intel-gen11', 'gpu-intel-gen12', 'gpu-intel-pvc']
+# https://github.com/intel/llvm/issues/15461
+# Wait for cpu driver uplifting
+config.unsupported_features += ['arch-x86_64']
 
-# GPU testing requires level_zero
-if 'opencl:gpu' in config.sycl_devices:
-   config.required_features += ['level_zero']
+# https://github.com/intel/llvm/issues/15953
+config.unsupported_features += ['arch-intel_gpu_acm_g12']
```

That's not gen12. It should be something like
```suggestion
config.unsupported_features += ['gpu-intel-gen12']
```

---

You can see from https://github.com/intel/llvm/issues/15461 that, @sarnex told me 'gpu-intel-gen12' will disable cpu test as well, and @aelovikov-intel said I should use device specific config like "arch-xxxx".

---

I see! Good point. In that case, I believe the following should suffice:
```suggestion
config.unsupported_features += ['arch-intel_gpu_12_0_0']
```

---

@steffenlarsen Done